elements/lisk-chain/src/block_reward.ts

```diff
@@ -121,3 +125,94 @@ export const calculateSupply = (
 
 	return supply;
 };
+
+export const getTotalRewardAndFees = (
+	blockInstance: BlockInstance,
+): { readonly totalFee: bigint; readonly totalBurnt: bigint } => {
+	// tslint:disable-next-line no-let
+	let totalFee = BigInt(0);
+	// tslint:disable-next-line no-let
+	let totalBurnt = BigInt(0);
+
+	for (const transaction of blockInstance.transactions) {
+		const minFee = BigInt(0); // TODO: It should get from transaction
+		totalFee += transaction.fee - minFee;
+		totalBurnt += minFee;
+	}
```

Comment:
Although I am ok with for loop here I think we prefer a more functional approach. 

```js
return blockInstance.transactions.reduce((acc, transaction) => {
   acc.totalFee = acc.totalFee + transaction.fee - minFee;
   acc.totalBurnt = acc.totalBurnt + minFee;
   return acc; 
}, {totalFee: 0, totalBurnt: 0})
```